packages/venia-ui/lib/queries/getCategory.graphql

```diff
@@ -21,22 +21,11 @@ query category(
         sort: $sort
     ) {
         items {
-            # Once graphql-ce/1027 is resolved, use a ProductDetails fragment here instead.
-            __typename
-            description {
-                html
-            }
+            # id is always required, even if the fragment includes it.
             id
-            media_gallery_entries {
-                id
-                label
-                position
-                disabled
-                file
-            }
-            meta_title
-            meta_keyword
-            meta_description
```

Comment:
I think this answers a previous question. The category page is no longer pre-fetching product details?

It's fine, I just want to make sure I understand how the app is working now.

Reply:
I found it & understand.